plugin.video.massengeschmack/addon.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<addon id="plugin.video.massengeschmack" name="Massengeschmack" version="2.2.0" provider-name="phoerious">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <language>en_GB de_DE</language>
```

Comment:
this should be kept as "en de". The changes I requested are only applied for the language files (attributes for the summary, description and disclaimer).